pkg/endpoint/bpf.go

```diff
@@ -218,12 +218,9 @@ func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
 	}
 	fw.WriteString(" */\n\n")
 
-	// If there hasn't been a policy calculated yet, we need to be sure we drop
-	// all packets, but only if policy enforcement
-	// is enabled for the endpoint / daemon.
-	if !e.PolicyCalculated &&
-		(e.Opts.IsEnabled(OptionIngressPolicy) || e.Opts.IsEnabled(OptionEgressPolicy)) &&
-		owner.PolicyEnforcement() != NeverEnforce {
+	// If policy has not been derived or calculated yet, all packets must
+	// be done until we have determined the policy of the endpoint.
```

Comment:
nit: `done` --> `dropped`

Reply:
yikes :) Ian got there before me

Reply:
Quick question: This enforces dropping independent of whether the endpoint might actually end up not having a matching policy i.e we might drop traffic initially for a `endpoint` which does not have any matching `ingress` or `egress` policy after the policy is finally calculated? Wondering if that's the intended behavior here?

Reply:
Nevermind, just saw your commit message. looks good.